etc/skate-style/base.mss

```diff
@@ -26,7 +26,7 @@
 #landuse[zoom>12] {
   [type='cemetery']      { polygon-fill: @cemetery; }
   [type='college']       { polygon-fill: @school; }
-  [type='commercial']    { polygon-fill: @industrial; }
+  [type='commercial']    { polygon-fill: @commercial; }
```

Comment:
I struggled for a bit to figure out where commercial areas were on the map - turned out this is because they were already styled the `industrial` color! I'm not sure if this is fully necessary since industrial and commercial are now the same, but may make it easier to understand where the different land types are. 

Alternatively, since now commercial, industrial, residential are all the same color, we could condense them into a single variable name.

Reply:
Yeah, thanks for that feedback! I added a middle-ground refactor for this PR - introducing `@background-grey` variable and having the existing `@industrial, @commercial, etc.` variables reference that instead of `#f4f4f5`.

This is in line with how the "halo" colors are configured [here](https://github.com/mbta/tile-server/blob/21b250514afcaa0861674227703788978fe06477/etc/skate-style/palette.mss#L126), which seems like a good mix of consistency & flexibility while these colors are still in testing.
> /* We set up a default halo color for places so you can edit them all
>    at once or override each individually. */